easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.24-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'LIBSVM'
+version = '3.24'
+
+homepage = 'https://www.csie.ntu.edu.tw/~cjlin/libsvm'
+description = """LIBSVM is an integrated software for support vector classification, (C-SVC, nu-SVC), regression
+ (epsilon-SVR, nu-SVR) and distribution estimation (one-class SVM). It supports multi-class classification."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://www.csie.ntu.edu.tw/~cjlin/libsvm/oldfiles/']
```

Comment:
I missed this easlier, but in existing easyconfigs for LIBSVM 3.24 we download from GitHub (https://github.com/cjlin1/libsvm), see #13076 .
Any reason not to keep doing that?